src/store/modules/order/OrderState.ts

```diff
@@ -0,0 +1,7 @@
+export default interface OrderState {
+    order: {
```

Comment:
https://github.com/hotwax/import/pull/5/files#r815722811

Reply:
Updated the order state